src/Auxiliary/Aux_Check_Conservation.cpp

```diff
@@ -103,7 +103,10 @@ void Aux_Check_Conservation( const char *comment )
    const bool GetPassiveSum      = ( PassiveNorm_NVar > 0 );
    const int  NVar_Flu           = NVar_NoPassive + NCOMP_PASSIVE + ( (GetPassiveSum)?1:0 );
    const int  idx_offset_flu     = 1;
-   const int  idx_offset_flu_com = idx_offset_flu+NVar_Flu;
+   const int  idx_offset_flu_com = idx_offset_flu + NVar_Flu;
+
+   int NStoredConRef_noTime_noPassive = 0;
+   NStoredConRef_noTime_noPassive += NVar_Flu - NCOMP_PASSIVE + 3;
```

Comment:
Add the comment "// +3: center-of-mass position" here and for the following two updates of `NStoredConRef_noTime_noPassive` as well.